src/test/java/org/broadinstitute/hellbender/engine/GATKToolUnitTest.java

```diff
@@ -349,12 +366,34 @@ public void testPicardIntervalList() throws Exception {
         tool.doWork();
 
         // ensure that the raw interval argument has not been expanded by Barclay, and that the post-merged
-        // intervals list contains 3 itervals (there are 4 in the file; 2 get merged)
+        // intervals list contains 3 intervals (there are 4 in the file; 2 get merged)
         Assert.assertEquals(tool.getIntervals().size(), 3);
 
         tool.onShutdown();
     }
 
+    @DataProvider(name = "commandLineExpansionExtensions")
+    private Object[][] commandLineExpansionExtensions() {
+        return new Object[][]{
+                {createTempFile("testExpansionSuppression", ".list")},
+                {createTempFile("testExpansionSuppression", ".args")},
+        };
+    }
+
+    @Test(dataProvider = "commandLineExpansionExtensions")
+    public void testPicardListExpansionSuppression(final File testFile) throws IOException {
+        final TestGATKToolWithSuppressedFileExpansion tool = new TestGATKToolWithSuppressedFileExpansion();
+        final CommandLineParser clp = new CommandLineArgumentParser(tool);
+        final String[] args = {
+                "--suppressedExpansionArg", testFile.getCanonicalPath(),
+        };
+        clp.parseArguments(System.out, args);
+        // file expansion is suppressed, so the list should contain the actual file name, not the contents
+        Assert.assertEquals(
+                tool.getSuppressedExpansionArg(),
+                Collections.singletonList(testFile.getCanonicalPath()));
```

Comment:
Add a second arg to your test tool that has expansion turned on, and prove that it gets expanded for both an `.args` file and a `.list` file in a separate test case.

Reply:
done.